cmd/cluster-kube-apiserver-operator/render/render.go

```diff
@@ -69,6 +70,7 @@ func NewRenderCommand() *cobra.Command {
 	cmd.Flags().StringVar(&renderOpts.manifest.configFileName, "manifest-config-file-name", "kube-apiserver-config.yaml", "The config file name inside the manifest-config-host-path.")
 	cmd.Flags().StringVar(&renderOpts.manifest.cloudProviderHostPath, "manifest-cloud-provider-host-path", "/etc/kubernetes/cloud", "A host path mounted into the apiserver pods to hold cloud provider configuration.")
 	cmd.Flags().StringArrayVar(&renderOpts.manifest.etcdServerURLs, "manifest-etcd-server-urls", []string{"https://127.0.0.1:2379"}, "The etcd server URL, comma separated.")
+	cmd.Flags().StringVar(&renderOpts.manifest.masterIpAddress, "manifest-master-ip-address", "192.168.124.11", "The IP address of the master0 host")
```

Comment:
If this is used only by etcd service, I will make it specific to etcd, like:

```
--manifest-etcd-service-ip
```

Also I would not hard-code the IP address here, but rather in installer?

Reply:
also maybe mark it as hidden or deprecated